apa102/src/main/java/com/google/android/things/contrib/driver/apa102/Apa102.java

```diff
@@ -67,7 +67,16 @@
     private Mode mLedMode;
 
     // RGB LED strip settings that have sensible defaults.
-    private int mLedBrightness = MAX_BRIGHTNESS >> 1; // default to half
+    private int mLedBrightnessGlobal = MAX_BRIGHTNESS/2;
```

Comment:
This was changed to division, while the ones below are still using shift. Perhaps revert this line?